docs/content/latest/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -6,9 +6,11 @@ headcontent: Tablespaces in YSQL
 image: /images/section_icons/secure/create-roles.png
 menu:
   latest:
-    identifier: explore-ysql-language-features-tablespaces
-    parent: explore-ysql-language-features
+    identifier: going-beyond-sql-tablespaces
```

Comment:
You're changing the identifier and parent in this file, but not in `tablespaces-platform.md` or `tablespaces-yugabyted.md` — is that on purpose?

Reply:
So those two files are included as part of **Cluster creation** tab and their URL's looks like: /latest/explore/ysql-language-features/going-beyond-sql/tablespaces/#yugabyted
/latest/explore/ysql-language-features/going-beyond-sql/tablespaces/#platform